src/App.js

```diff
@@ -1,12 +1,59 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import Card from "./components/Card";
+import axios from "axios";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [error, setError] = useState("");
+    const [data, setData] = useState([]);
+    const [offset, setOffset] = useState(0)
+    const [maxOffset, setMaxOffset] = useState(1);
+
+    useEffect(() => {
+        async function fetchData() {
+            setError("");
+            try {
+                const response = await axios.get(`https://pokeapi.co/api/v2/pokemon?offset=${offset}&limit=20`);
+                setData(response.data.results);
+                setMaxOffset(response.data.count);
+            } catch (e) {
+                console.log(e);
+                setError("Er is geen informatie gevonden...")
+            }
+        }
+
+        fetchData();
+    }, [offset]);
+
+    return (
+        <>
+            <header>
+                <img className="logo" src="https://raw.githubusercontent.com/PokeAPI/media/master/logo/pokeapi_256.png"
+                     alt="logo"/>
+                <nav>
+                    <button
+                        onClick={() => {
+                            setOffset(Math.max(0, offset - 20))
+                        }}
+                        disabled={offset === 0 && true}
```

Comment:
Deze kun je versimpelen
```suggestion
                        disabled={offset === 0}
```